src/Components/Server/src/ComponentHub.cs

```diff
@@ -220,7 +221,18 @@ public async ValueTask ReceiveByteArray(int id, byte[] data)
                 return;
             }
 
-            await circuitHost.ReceiveByteArray(id, data);
+            _ = circuitHost.ReceiveByteArray(id, data);
```

Comment:
Could you clarify why we don't want to await here? I would have thought we do want to await but I guess there must be a reason for the change. Would be good to add a code comment too since others might wonder in the future too.

Reply:
Update: I see from the description below we don't want to block the circuit, and this is simply consistent with the logic in all the other hub methods. So it doesn't need a special comment about this of its own.